examples/node_reputation/node_reputation-server/main.go

```diff
@@ -0,0 +1,39 @@
+// Copyright (C) 2018 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package main
+
+import (
+	"fmt"
+	"net"
+
+	"google.golang.org/grpc"
+	nodereputation "storj.io/storj/pkg/node_reputation"
+	proto "storj.io/storj/protos/node_reputation"
+)
+
+func main() {
+	db, err := nodereputation.SetServerDB("./Server.db")
+	if err != nil {
+		fmt.Println("err")
+	}
+
+	lis, err := net.Listen("tcp", fmt.Sprintf(":%d", 7777))
+	if err != nil {
+		fmt.Println("err")
+	}
+
+	s := nodereputation.Server{}
+
+	grpcServer := grpc.NewServer()
+
+	proto.RegisterNodeReputationServer(grpcServer, &s)
+
+	if err := grpcServer.Serve(lis); err != nil {
+		fmt.Println("err")
+	}
+
+	defer grpcServer.GracefulStop()
```

Comment:
generally if your statements are the last statements in the function, it makes no difference if they're deferred or not. i don't think you need the deferred keyword here. defer is for putting closing logic up by where it's created, so i'd export more something like

```
db, err := nodereputation.SetServerDB(*serverPath)
if err != nil {
  log.Fatal(err)
}
defer db.EndServerDB()

// ...

grpcServer := grpc.NewServer()
defer grpcServer.GracefulStop()

proto.RegisterNodeReputationServer(grpcServer, &s)
if err := grpcServer.Serve(lis); err != nil { 
// ...

```

also, i'm surprised `s` is an empty struct. shouldn't it have some state in it or something?